pgjdbc/build.gradle.kts

```diff
@@ -210,7 +210,6 @@ tasks.shadowJar {
 
 tasks.register<Jar>("cleanShadowJar") {
 
-    archiveClassifier.set("all")
```

Comment:
What is it for?

Reply:
Currently, when it builds the file name is `aws-postgresql-jdbc-0.1.0-all.jar`
Removing this line just removes the `-all` part in the name.

Also found a bug now that this was brought up. The draft release is now missing the file, but the Maven release & local still has it.